v3/lints/cabf_ev/lint_ev_orgid_inconsistent_subj_and_ext.go

```diff
@@ -0,0 +1,114 @@
+/*
+ * ZLint Copyright 2024 Regents of the University of Michigan
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+/*
+ * Contributed by Adriano Santoni <adriano.santoni@staff.aruba.it>
+ * of ACTALIS S.p.A. (www.actalis.com).
+ */
+
+package cabf_ev
+
+import (
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/v3/lint"
+	"github.com/zmap/zlint/v3/util"
+
+	"errors"
+	"regexp"
+)
+
+func init() {
+	lint.RegisterCertificateLint(&lint.CertificateLint{
+		LintMetadata: lint.LintMetadata{
+			Name:          "e_ev_orgid_inconsistent_subj_and_ext",
+			Description:   "Checks that the organizationIdentifier Subject attribute and the CABFOrganizationIdentifier extension are consistent",
+			Citation:      "EVGs 9.2.8 and 9.8.2",
+			Source:        lint.CABFEVGuidelines,
+			EffectiveDate: util.CABFEV_Sec9_2_8_Date,
+		},
+		Lint: NewOrgIdInconsistentSubjAndExt,
+	})
+}
+
+// According to EVGs 9.2.8
+type OrganizationIdentifier struct {
+	Scheme    string
+	Country   string
+	State     string
+	Reference string
+}
+
+func ParseOrgId(orgIdString string, orgId *OrganizationIdentifier) error {
+
+	// This is according to the EVG (stricter than ETSI EN 319 412-1)
+	OrgIdPattern := `^[A-Z]{3}[A-Z]{2}(?:\+[A-Z]{2})?\-.+$`
+
+	compiledRegexp, err := regexp.Compile(OrgIdPattern)
+	if err != nil {
+		// This should neve occur, but one never knows....
+		panic(err)
```

Comment:
[regexp.MustCompile](https://pkg.go.dev/regexp#MustCompile) will do what you're looking for here.

It is also common to place this regex compilation in the outer scope...

```go
var compiledRegexp = regexp.MustCompile(`^[A-Z]{3}[A-Z]{2}(?:\+[A-Z]{2})?\-.+$`)

func ParseOrgId(orgIdString string, orgId *OrganizationIdentifier) error {
    ....
}
```

Doing so makes it such that any panic that gets introduced would get triggered merely by starting up the program, thus reducing the likelihood of it making past CICD as code coverage on the line is guaranteed,